src/api/create-vis.js

```diff
@@ -12,6 +12,7 @@ var DEFAULT_OPTIONS = {
   tooltip: true, // TODO: it seems that this is no longer used
   logo: true,
   show_empty_infowindow_fields: false,
+  showLimitErrors: true,
```

Comment:
by default this should be `false`

Reply:
yep, my bad, I was using it for testing purposes, let me fix it!